std/file.d

```diff
@@ -1585,6 +1675,31 @@ if (isConvertibleToString!R)
     return timeLastModified!(StringTypeOf!R)(name);
 }
 
+///
+@safe unittest
+{
+    import std.datetime : abs, DateTime, hnsecs, SysTime;
+
+    scope(exit) deleteme.remove;
+    write(deleteme, "a");
+
+    SysTime accessTime = SysTime(DateTime(2010, 10, 4, 0, 0, 30), 0.hnsecs);
+    SysTime modificationTime = SysTime(DateTime(2011, 10, 4, 0, 0, 30), 10.hnsecs);
+    setTimes(deleteme, accessTime, modificationTime);
+
+    SysTime accessTimeResolved, modificationTimeResolved;
+    getTimes(deleteme, accessTimeResolved, modificationTimeResolved);
+
+    assert(accessTime == accessTimeResolved);
+    assert(modificationTime == modificationTimeResolved);
+
+    import std.datetime : Clock, seconds;
```

Comment:
Everything before this line isn't related to `timeLastModified` and should be removed IMO